features/authentication.feature

```diff
@@ -117,6 +117,25 @@ Feature: OAuth authentication
     Then the output should not contain "github.com password for mislav"
     And the file "../home/.config/hub" should contain "oauth_token: OTOKEN"
 
+  Scenario: Credentials from GITHUB_TOKEN
+    Given the GitHub API server:
+      """
+      post('/authorizations') {
```

Comment:
`POST /authorizations` should never happen in this case. There is no auth phase. We provided an existing token.